src/App.svelte

```diff
@@ -17,12 +17,22 @@
 		isLoggedIn = status;
 	});
 
+	let eventStatus: boolean;
+	let showEvents: boolean;
+
 	export let messages: messageType[] = [];
 
 	// truncate the number of messages show in the app
 	$: if (messages.length >= 50) {
 		messages = messages.slice(1);
 	}
+	
+	$: eventStatus = showEvents;
+
+	$: if(eventStatus){
```

Comment:
We don't want the event stream to stop, just auto follow be disabled for a period.